userData&Order.c

```diff
@@ -0,0 +1,36 @@
+//
+// Created by Victor on 11/3/2019.
+//
+#include <stdio.h>
+#include <string.h>
+#define MAX_FOOD_NAME 10
+#define MAX_SPECIALITY_NAME 11
+#define MAX_NR_FOODS 3
+#define MAX_NR_OF_PRICES 3
+#define MAX_DRINK_NAME 10
+
+void signIn(char Username[], char Password[]) {
+
+    printf("Welcome to Food Thingies!\n");
+    printf("Please sign in to continue\n");
+    printf("---Username:\n>");
+    gets(Username);
+    printf("---Password:\n>");
+    gets(Password);
+}
+
+void printUserOrder(char Username[], char foodOptions[][MAX_FOOD_NAME], int foodsChoice, char foods[MAX_NR_FOODS][MAX_NR_FOODS][MAX_SPECIALITY_NAME],
+                    int specialityChoice, char YesNo[][4], int prices[MAX_NR_FOODS][MAX_NR_OF_PRICES], char drink[][MAX_DRINK_NAME],
```

Comment:
you are sending more data then needed... you don't have to send the whole food options, just the chosen food.. same applies for other parameters. If you send only the data you need, you might also have less parameters (cause you're having quite a lot now)